android/src/main/java/vn/luongvo/kmm/survey/android/ui/common/CloseButton.kt

```diff
@@ -0,0 +1,39 @@
+package vn.luongvo.kmm.survey.android.ui.common
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import vn.luongvo.kmm.survey.android.R
+
+@Composable
+fun CloseButton(
+    onClick: () -> Unit,
+    modifier: Modifier
```

Comment:
For modifier, we should have default value for it
```suggestion
    modifier: Modifier = Modifier
```